src/pages/wallet/Transfer.vue

```diff
@@ -265,14 +274,82 @@ function convertFiatToCrypto(fiatValue = 1) {
   return fiatValue / (selectedToken.value?.price?.[selectedFiat] || 1);
 }
 
-const generateTransaction = async (amount: number): Promise<Transaction> => {
+const createSolTransactionInstruction = (amount: number) => {
+  return [
+    SystemProgram.transfer({
+      fromPubkey: new PublicKey(ControllerModule.selectedAddress),
+      toPubkey: new PublicKey(resolvedAddress.value),
+      lamports: amount * LAMPORTS_PER_SOL,
+    }),
+  ];
+};
+
+interface CreateTransferFields {
+  recipient: Recipient;
+  amount: number;
+  splToken?: SPLToken;
+  reference?: References;
+  memo?: Memo;
+}
+
+async function createTransfer(
```

Comment:
we could recreate the VersionedTransaction from the Transaction.
Copy the code and modify it might not be good idea. ( should be last resort )

Reply:
done